src/main/java/org/apache/maven/plugins/war/packaging/AbstractWarPackagingTask.java

```diff
@@ -342,14 +344,16 @@ protected boolean copyFile( WarPackagingContext context, File source, File desti
     {
         context.addResource( targetFilename );
 
-        if ( onlyIfModified && destination.lastModified() >= source.lastModified() )
+        BasicFileAttributes readAttributes = Files.readAttributes( source.toPath(), BasicFileAttributes.class );
+        if ( onlyIfModified && destination.lastModified() >= readAttributes.lastModifiedTime()
+                .toMillis() )
```

Comment:
What is the advantage to use another API here?

Reply:
It is okay for me. I didn't know about it, and it looked a bit harder to read for me at first, but if it is NIO, then big +1.